wallet/udb/addressmanager.go

```diff
@@ -482,31 +482,31 @@ func (m *Manager) GetMasterPubkey(ns walletdb.ReadBucket, account uint32) (strin
 // keys for it and track the state of the internal and external branches.
 //
 // This function MUST be called with the manager lock held for writes.
-func (m *Manager) loadAccountInfo(ns walletdb.ReadBucket, account uint32) (*accountInfo, error) {
+func (m *Manager) loadAccountInfo(ns walletdb.ReadBucket, account uint32) (*accountInfo, *dbBIP0044AccountRow, error) {
 	// Return the account info from cache if it's available.
 	if acctInfo, ok := m.acctInfo[account]; ok {
-		return acctInfo, nil
+		return acctInfo, nil, nil
```

Comment:
If this change is made, it should not return a nil BIP0044 account row without error.  Code below dereferences row without any nil pointer checks.